src/components/LocationAutocompleteInput/GeocoderGoogleMaps.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
-import { getPlacePredictions, getPlaceDetails } from '../../util/googleMaps';
-
+import { getPlacePredictions, getPlaceDetails, locationBounds } from '../../util/googleMaps';
+import { userLocation } from '../../util/maps';
 import css from './LocationAutocompleteInput.css';
 
+export const CURRENT_LOCATION_ID = 'current-location';
+const CURRENT_LOCATION_BOUNDS_DISTANCE = 1000; // meters
```

Comment:
Should this be placed in config.js? Then marketplace customizer could easily modify it.

Reply:
I think we have to think about the whole maps config when we can consider the Mapbox support in its entirety. I'd keep these a bit hidden for now, especially since part of configuring the default searches would still be in the geocoder file. Splitting that into several places wouldn't be any better.

We already have some Google Maps specific options in the config. We should maybe have separate keys for Google Maps and Mapbox that contain all the config for those map providers. So let's not hurry now and add random partial setting to the global config...

How does this sound?

Reply:
Sounds good, let's see how it looks when Mapbox is in place.